tests/test_dask_image/test_ndmeasure/test_find_objects.py

```diff
@@ -0,0 +1,40 @@
+import dask.array as da
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+import pytest
+
+import dask_image.ndmeasure
+
+
+@pytest.fixture
+def label_image():
```

Comment:
Separately, it would also be good to construct a really large test example (not to keep/run in the tests, just to give this thing a whirl and see what it can do)

Reply:
Going to release this into the wild and see if anyone out there will end up road testing this for us (I've left it languish here too long while I've been meaning to get around to it)

Reply:
Issue made here to track any future performance testing of `find_objects` https://github.com/dask/dask-image/issues/252